README.md

```diff
@@ -60,13 +60,26 @@ const stripe = Stripe('sk_test_...');
 //…
 ```
 
+### Usage with TypeScript
 
-Or using TypeScript:
+Stripe does not currently maintain typings for this package, but there are 
+community typings available from DefinitelyTyped. 
+
+To install:
+
+```sh
+npm install --dev @types/stripe
+```
+
+To use:
 
 ```ts
+// Note `* as` and `new Stripe` for TypeScript:
 import * as Stripe from 'stripe';
 const stripe = new Stripe('sk_test_...');
-//…
+
+// Types are exposed under the `StripeNode` namespace:
```

Comment:
The `StripeNode` namespace is old information since the Stripe types package rebranded - types and interfaces are now explicitly exported from the types package :)

Here's example usage:

```
import * as Stripe from 'stripe'

const stripe = new Stripe('');

const subscription: Promise<Stripe.subscriptions.ISubscription> = stripe.subscriptions.create({ ... })
```

or, just pulling in types without the whole Stripe package

```
import { subscriptions } from "stripe";

const updateSubscription = (sub: subscriptions.ISubscription) => { ... }
```

Reply:
Ah, thanks for catching that!